packages/babel-generator/src/buffer.js

```diff
@@ -40,7 +39,7 @@ export default class Buffer {
 
     const map = this._map;
     const result = {
-      code: trimEnd(this._buf.join("")),
+      code: this._trimEnd(this._buf.join("")),
```

Comment:
As a new contributor I'm not really equipped to answer this question. Trailing whitespace doesn't seem important, but I guess there could be misbehaving plugins that introduce large amounts of it.